packages/segmentedControl/README.md

```diff
@@ -0,0 +1,13 @@
+# SegmentedControl
+
+A segmented control is group of 2 or more buttons users can select a value from. A segmented control can be a good alternative to a dropdown or a select input because it requires less interactions to make a selection. A segmented control can be a good alternative to a group of radio buttons when there are space limitations or form controls would look visually awkward.
```

Comment:
just out of curiosity: what's the rationale behind not calling it a "Button Group" like e.g. Bootstrap and Foundation do?

Reply:
I think of button groups as just being a styled list of buttons that have related actions.
This component is a list of options a user chooses from. I think of it more like a fancy group of radio buttons.